doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_architecture_pm.rst

```diff
@@ -153,100 +148,22 @@ Instead, their state is retained by software in RAM.
-The state of these peripherals is not retained by hardware and must be restored by software before the peripheral is activated.
-
-See the following sections for the lists of peripheral groups and the related software modules responsible for restoring the peripheral's state for each group.
-
-Peripherals in local domains
-============================
-
-All local domains include a common set of hardware modules.
-In addition to these, most local domains also contain domain-specific peripherals.
-
-Common peripherals for all local domains
-----------------------------------------
-
-Each local domain contains a set of peripherals that are classified consistently across all local domains.
-The following table summarizes the active peripherals that need handling when exiting the *Local Suspend to RAM* state.
-
-+---------------+--------------------+--------------------+--------------------+--------------------------+
-|Type           | List of the        | Source of data to  | Time of restoration| Software module          |
-|               | peripherals        | restore            |                    | responsible for restoring|
-+===============+====================+====================+====================+==========================+
-|Active         | * ``MVDMA``        | Device driver's    | Decided by the     | The device driver        |
-|peripherals    |                    | code and data      | driver             |                          |
-+---------------+--------------------+--------------------+--------------------+--------------------------+
-
-Peripherals specific to the Application Domain
-----------------------------------------------
-
-There are no peripherals specific to the Application Domain.
-
-Peripherals specific to the Secure Domain
------------------------------------------
-
-The Secure Domain contains additional peripherals that require handling in the *Local Suspend to RAM* state.
-
-+---------------+--------------------+--------------------+--------------------+--------------------------+
-|Type           | List of the        | Source of data to  | Time of restoration| Software module          |
-|               | peripherals        | restore            |                    | responsible for restoring|
-+===============+====================+====================+====================+==========================+
-|Active         | * ``CRACEN``       | Device driver's    | Decided by the     | The device driver        |
-|peripherals    |                    | code and data      | driver             |                          |
-+---------------+--------------------+--------------------+--------------------+--------------------------+
+Mapping application states to software power states
+***************************************************
 
-Peripherals specific to the Radio Domain
-----------------------------------------
+The following table maps application states to software power states:
```

Comment:
@hubertmis  Please check if the mapping in the table is correct

Reply:
Yes, I think that's good. It doesn't show all the details, but adding them would make it hard to read